test/claims-status/utils/helpers.unit.spec.js

```diff
@@ -556,4 +557,19 @@ describe('Disability benefits helpers: ', () => {
       expect(alertContent.type).to.exist;
     });
   });
+
+  describe('categorizeIssues', () => {
+    it('returns an array of same length as input array', () => {
+      const { issues } = mockData.data[1].attributes;
+      const formattedIssues = categorizeIssues(issues);
+      expect(formattedIssues.length).to.equal(issues.length);
+    });
+
+    it('returns an array of objects with status and description', () => {
+      const { issues } = mockData.data[1].attributes;
+      const formattedIssues = categorizeIssues(issues);
+      expect(formattedIssues[0].status).to.exist;
```

Comment:
Would it be better to test all the issues with something like the following?
```js
expect(formattedIssues.every(i => i.status && i.description)).to.be.true;
```

Reply:
That seems like a good idea. Instead of using something like ^ I'll use exactly ^ 😛